README.md

```diff
@@ -1 +1,50 @@
-# Awesome-Books-Store
\ No newline at end of file
+# Awesome Books
+
+A Website demo for our project of book store, The website has ability of adding and removing you books from yor library, Thats reflects our coding skills.
+
+
+## Live Link
+
+[Awesome Books](https://banstein.github.io/Awesome-Books-Store/)
+
+## Built With!
+
+- HTML
+- CSS
+- JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+clone repo: https://github.com/Banstein/Awesome-Books-Store.git
+
+### Authors
+
+👤 **Author1**
+
+- GitHub: [@Banstein](https://github.com/Banstein)
+- Twitter: [@islam_bahnas](https://twitter.com/islam_bahnas)
+- LinkedIn: [islam-bahnas](www.linkedin.com/in/islam-bahnas)
+
+👤 **Author2**
+
+- GitHub: [@jerryowusu](https://github.com/jerryowusu)
+- Twitter: [@jerryowusu11](https://twitter.com/jerryowusu11)
+- LinkedIn: [Jerry Owusu](https://www.linkedin.com/in/jeremiah-owusu-b50a70173/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed
```

Comment:
- [ ] the link to the License is not working as intended, please have a look at [the Microverse](https://github.com/microverseinc/readme-template) template for an example of how it is supposed to work.

Reply:
Resolved. Thanks